modules/eks/cluster/github-actions-iam-policy.tf

```diff
@@ -55,19 +39,11 @@ data "aws_iam_policy_document" "github_actions_iam_policy" {
   statement {
     effect = "Allow"
     actions = [
-      "ssm:GetParameters"
-    ]
-    resources = [
-      "arn:aws:ssm:*:*:parameter/platform/${module.eks_cluster.eks_cluster_id}/*"
-    ]
-  }
-  statement {
-    effect = "Allow"
-    actions = [
+      "ssm:GetParameters",
       "ssm:GetParametersByPath"
     ]
     resources = [
-      "*"
+      "arn:aws:ssm:*:*:parameter/platform/${module.eks_cluster.eks_cluster_id}/*"
```

Comment:
Can we replace `platform` with `module.this.tenant` if available?

Reply:
I want to get rid of this whole thing. It should be pulled in as a mixin/override, and I don't understand its use case. Meanwhile, I don't want to mess with it.

Reply:
this was originally a mixin for a client, so it doesnt need to be upstream. We should remove this file and the related mixin